scripts/install

```diff
@@ -53,7 +53,7 @@ sed -i "s@YNH_ADMIN_USER@$admin@g" ../conf/dokuwiki.php
 sudo mkdir "$final_path"
 ynh_app_setting_set $app final_path $final_path
 
-SETUP_SOURCE "dokuwiki-c5525093cf2c4f47e2e5d2439fe13964.tgz"
+SETUP_SOURCE "dokuwiki-stable.tgz"
```

Comment:
This URL is not "absolute", which is problematic in my opinion --> at the next Dokuwiki release, the archive content will be changed to the newer version so further package installations will install this newer version, which may have not been tested by the team, and could require package adaptations.
Maybe could you find out which daily archive corresponds to that official release and point specifically at it?